src/components/molecules/forms/settingsForm/SettingsForm.tsx

```diff
@@ -56,6 +58,11 @@ const SettingsForm: React.FC<Props> = ({ init }) => {
     try {
       await updateMember(data);
       history.push('/profile');
+      addToast({
+        title: 'Suksess',
```

Comment:
Sometimes we use the norwegian `Suksess`, and sometimes we use `Success`. We should stick to one of them and not mix

Reply:
Norwegian allerts/toasts seems to be what mostly used hence  we should use norwegian?

Reply:
I don't think I have a preference really as long as they are consistent🤷‍♂️